changelog.d/15515.misc

```diff
@@ -0,0 +1 @@
+Updated all Inline SQL queries using boolean parameters as supported in sqlite 3.27
```

Comment:
```suggestion
Updated all SQL queries to inline boolean parameters as supported in sqlite 3.27.
```

Reply:
Continued in https://github.com/matrix-org/synapse/pull/15525#pullrequestreview-1515433288